protocols/network.md

```diff
@@ -52,16 +52,16 @@ With one or more IP addresses for the host the client wants to contact, it
 sends a "connect request". The connection it wants to establish is called a
 TCP ([Transmission Control
 Protocol](https://en.wikipedia.org/wiki/Transmission_Control_Protocol)) or
-[QUIC](https://en.wikipedia.org/wiki/QUIC) connection and they work similar to
+[QUIC](https://en.wikipedia.org/wiki/QUIC) connection, which is like
 connecting an invisible string between two computers. Once established, the
 string can be used to send a stream of data in both directions.
 
 If the client has received more than one address for the host, it will
-traverse that list of addresses when connecting and in case one of fails it
-will try to connect to the next one until either one connect works or they all
-fail.
+traverse that list of addresses when connecting, and if one address fails it
+will try to connect to the next one, repeating until either one address works 
+or they've all failed.
```

Comment:
A contraction to remove. The CI job didn't detect it, but it will going forward.

Reply:
done